app/templates/route-components/assets.hbs

```diff
@@ -124,6 +124,7 @@
 		{{icon-box name='community' size='small'}}
 		{{icon-box name='eye' size='small'}}
 		{{icon-box name='eye-crossed' size='small'}}
+		{{icon-box name='fandom-heart' size='small'}}
```

Comment:
I was never 100% sure about it, but this icon looks like ["Company Assets"](http://wikiadesignsystem.com/#/identity/assets). Shouldn't it be placed there?